pyproject.toml

```diff
@@ -0,0 +1,8 @@
+[project]
+name = "streamspy"
+requires-python = ">=3.10"
+dynamic = ["version", "description"]
+
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
```

Comment:
not sure why I decided to only now start packaging things up into a pyproject.toml but probably the right thing to do